staging/src/k8s.io/apimachinery/pkg/labels/selector.go

```diff
@@ -883,8 +882,6 @@ func SelectorFromSet(ls Set) Selector {
 			return internalSelector{}
 		}
 	}
-	// sort to have deterministic string representation
```

Comment:
The comment here explicitly describes why this sorting was done. Please don't remove this without understanding the reason for it.

Reply:
The string representation is used by unit tests.

To get deterministic comparison result in unit test, the sorting can be done in the test.
That's better than every caller pays the price of sorting.

Reply:
It's also used externally and for string serialization. We should not change the behavior of these methods.